test/unit.h

```diff
@@ -62,6 +62,10 @@ void print_buffer(const char *desc, const unsigned char *buffer, size_t len);
 #define TEST_DECL(func, data)        { #func, func, data, 0, 0, 0 }
 #endif
 
+#ifndef CERTS_DIR
+#define CERTS_DIR "./certs/"
```

Comment:
directory path fragments in vars/macros shouldn't include the trailing slash -- the slashes should be added in the course of assembling the full path.  that way, bad values for the var/macro lead to immediate "not a directory" errors, rather than mysterious later failures.  it's also more readable code.

btw, on Linux and BSD at least, multiple slashes work like one:
```
$ ls -l /bin/////ls
-r-xr-xr-x  1 root wheel 34408 Nov 10  2023 /bin/////ls
```

Reply:
The trailing slash is on there, because if "CERTS_DIR" is not defined, I wanted the path to be in the current working directory. Otherwise the path would be at the root directory which would definitely cause issues.

Reply:
Additionally, I would argue `CERTS_DIR "/filename.ext"` is more readable than `CERTS_DIR "filename.ext"`. The missing slash would be an eye sore for me.